fms/utils/generation.py

```diff
@@ -245,3 +243,21 @@ def truncate_after_eos(
         index = eos_index[0]
         result = result[: index + 1]
     return result
+
+
+def trim_prefix(
+    result: torch.Tensor, pad_token_id: Optional[int] = None
+) -> torch.Tensor:
+    """
+    Helper function to return a trimmed sequence of token IDs where
+    all padding tokens (always 0 on our code) are removed.
+    """
+    if pad_token_id is None:
+        pad_token_id = 0
```

Comment:
why not just `pad_token_id: int = 0` in the signature instead of None and setting here?